nano/node/voting.cpp

```diff
@@ -184,7 +184,7 @@ nano::vote_generator::~vote_generator ()
 	stop ();
 }
 
-void nano::vote_generator::process (store::write_transaction const & transaction, nano::root const & root_a, nano::block_hash const & hash_a)
+bool nano::vote_generator::process (store::write_transaction const & transaction, nano::root const & root_a, nano::block_hash const & hash_a)
```

Comment:
After those modification a better suiting name for this method `should_vote`, probably can be made const too.